honeybee_grasshopper_radiance/src/HB Daylight Factor.py

```diff
@@ -180,49 +174,35 @@ def ToString(self):
         msg = 'Input _model contains no sensor grids, which will result in a ' \
             'meaningless simulation.\nMake sure that you have assigned grids to ' \
             'the Model with the "HB Assign Grids and Views" component.'
-        give_warning(ghenv.Component, msg)
+        raise ValueError(msg)
```

Comment:
Also, if you are changing this to an error, you can remove the import of `give_warning` at the top since it's not longer used. (same goes for the other recipe).